src/terra-dev-site/doc/application/how-to.2/theme-strategy.doc.mdx

```diff
@@ -0,0 +1,166 @@
+
+`example-component/ExampleComponent.module.scss`
+
+```scss
+// The values in this file represent the default theme. The default theme is applied using the default values in the var() function.
+:local {
+  .example-component {
+    background-color: var(--terra-application-example-component-background-color, #fff);
+    color: var(--terra-application-example-component-color, inherit);
+    font-size: var(--terra-application-example-component-font-size, 1rem);
+  }
+}
+```
+
+To add themes, create new files for each theme and include them in the original SCSS file. Theme files define variables that are applied when the theme becomes active. Themes must be locally scoped and must use a supported theme class name.
+
+`example-component/ExampleComponent.orion-fusion-theme.module.scss`
+
+```scss
+:local {
+  .orion-fusion-theme {
+    --terra-application-example-component-background-color: #f6f8fa;
+    --terra-application-example-component-color: rgb(36, 41, 46);
+    --terra-application-example-component-font-size: 2rem;
+  }
+}
+```
+
+`example-component/ExampleComponent.clinical-lowlight-theme.module.scss`
+
+```scss
+:local {
+  .clinical-lowlight-theme {
+    --terra-application-example-component-background-color: rgb(34, 42, 46);
+    --terra-application-example-component-color: rgb(178, 181, 182);
+    --terra-application-example-component-font-size: 1rem;
+  }
+}
+```
+
+Include the new theme files in the original SCSS file.
+
+`example-component/ExampleComponent.module.scss`
+
+```scss
+// Import each theme.
+@import './ExampleComponent.clinical-lowlight-theme.module';
+@import './ExampleComponent.orion-fusion-theme.module';
+
+:local {
+  .example-component {
+    background-color: var(--terra-application-example-component-background-color, #fff);
+    color: var(--terra-application-example-component-color, inherit);
+    font-size: var(--terra-application-example-component-font-size, 1rem);
+  }
+}
+```
+
+The recommended file structure ends up looking like this:
+
+```
+.
+├── example-component
+│   ├── ExampleComponent.clinical-lowlight-theme.module.scss
+│   ├── ExampleComponent.jsx
+│   ├── ExampleComponent.module.scss
+│   └── ExampleComponent.orion-fusion-theme.module.scss
+```
+
+## Webpack
+
+To enable themes, teams must define a terra-theme.config.js configuration file. This file is read by the [terra-toolkit postcss loader](https://github.com/cerner/terra-toolkit/tree/master/config/webpack/postcss) to aggregate themes. This file must be defined for theme CSS assets to be included in the webpack bundle. The loader runs automatically for webpack configurations using terra-toolkit's webpack config.
+
+```js
+const themeConfig = {
+  theme: 'terra-dark-theme', // The default theme to be enabled on page load.
+  scoped: ['terra-light-theme', 'terra-lowlight-theme'], // An array of scoped themes. Note: Scoped themes do not work in IE 10.
+};
+
+module.exports = themeConfig;
+```
+
```

Comment:
should we add a testing section outlining our script and unit test expectations?

Reply:
This would be a good addition for this guide, do we have existing documentation for this functionality for reference?

Reply:
I added a small section for testing, there wasn't a lot of documentation to draw upon so any additional suggestions are welcome.